### PR TITLE
Update to version 1.7.2

### PR DESCRIPTION
**Pull request by afarchy**

### CODE REVIEW

**Review by milosgacanovic — APPROVED** [review]

**Review by milosgacanovic — APPROVED** [review]

File: Runtime/ApiRequests.cs [hunk 1]
```diff
@@ -68,11 +68,11 @@ public static async Task<GetImagineResult> GetRequestStatusAsync(string imagineO
             return await GetAsync<GetImagineResult>("imagine/requests/obfuscated-id/" + imagineObfuscatedId, apiKey);
         }
 
-        public static async Task<Texture2D> DownloadTextureAsync(string textureUrl)
+        public static async Task<Texture2D> DownloadTextureAsync(string textureUrl, bool readable = false)
```
Comment (StephenHodgson) [inline Runtime/ApiRequests.cs:71]: imho this should probably always be true in our case

File: Scenes/SkyboxScene.unity [hunk 2]
```diff
@@ -18059,7 +18059,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 80c37ee5440861a47884a14ba990aa23, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  _apiKey: 
+  _apiKey: QmNFgVOP9Fm7l38AW0YbtMdetF46i7jYhS9ii0a7Hlyf5KzrXzCK8rYUeYVt
```
Comment (StephenHodgson) [inline Scenes/SkyboxScene.unity:18062]: Might wanna get rid of this and renew your api key,

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
Remove API key

File: Runtime/ApiRequests.cs (hunk 1 above)
Comment (afarchy) [reply Runtime/ApiRequests.cs:71]: It's not generally needed at runtime when we can use a compute shader

File: Runtime/BlockadeLabsSkyboxGenerator.cs [hunk 3]
```diff
@@ -504,9 +504,13 @@ private async Task<GetImagineResult> PollForResultAsync(string imagineObfuscated
 
         private Task WaitForSeconds(float seconds)
         {
-            var tcs = new TaskCompletionSource<object>();
-            StartCoroutine(WaitForSecondsEnumerator(tcs, seconds));
-            return tcs.Task;
+            #if UNITY_EDITOR
+                return Task.Delay((int)(seconds * 1000));
```
Comment (StephenHodgson) [inline Runtime/BlockadeLabsSkyboxGenerator.cs:508]:
```suggestion
                return Task.Delay((int)(seconds * 1000)).ConfigureAwait(true);
```

You could also use editor coroutines and use EditorWaitForSecondsRealtime

**Pull request merged**